taurus/entity/object/tests/test_material_run.py

```diff
@@ -59,6 +59,9 @@ def test_process_run():
     copy_material = loads(dumps(material_run))
     assert dumps(copy_material) == dumps(material_run)
 
+    assert 'output_material' in process_run.__repr__()
```

Comment:
You could write this as `repr(process_run)`.

Reply:
Done